modules/download_to_pc.R

```diff
@@ -66,7 +68,8 @@ download_pc_button_server <- function(id,
           # Write output to user
           write.table(data_to_download, file_csv, sep = ',', row.names = FALSE, append = F)
 
-          zip(zipfile=file, files=c(file_csv, file_readme))
+          log_trace("Creating of zipfile {file} started")
```

Comment:
Could you add the name of the module?
log_trace("mod dwnld pc: Creating of zipfile {file} started")

Reply:
In above log message the module name is already part of the log message, I added the module name in the last message, see https://github.com/rivm-syso/Analyse-Together/pull/244/commits/04419a21966d0b4016218459a134351e727221c2.